packages/ai/src/requests/stream-reader.ts

```diff
@@ -215,19 +224,11 @@ export function aggregateResponses(
               }
               newPart.text = part.text;
             }
-            if (part.functionCall) {
-              newPart.functionCall = part.functionCall;
-            }
-            if (Object.keys(newPart).length === 0) {
-              throw new AIError(
```

Comment:
Just for clarity, this part is intentional, if I am reading the iOS change correctly, we don't want to throw this in streaming.